src/core/search/search.cc

```diff
@@ -454,20 +455,30 @@ string_view Schema::LookupAlias(string_view alias) const {
   return alias;
 }
 
-FieldIndices::FieldIndices(Schema schema, PMR_NS::memory_resource* mr)
-    : schema_{std::move(schema)}, all_ids_{}, indices_{} {
+IndicesOptions::IndicesOptions() {
+  static absl::flat_hash_set<std::string> kDefaultStopwords{
+      "a",    "is",    "the",  "an",    "and",   "are",  "as",   "at", "be",  "but",  "by",
+      "for",  "if",    "in",   "into",  "it",    "no",   "not",  "of", "on",  "or",   "such",
+      "that", "their", "then", "there", "these", "they", "this", "to", "was", "will", "with"};
+
+  stopwords_ = kDefaultStopwords;
+}
+
+FieldIndices::FieldIndices(const Schema* schema, const IndicesOptions* options,
```

Comment:
if the ownership stays with the caller while this object lives, it's better using `const refs`